testsuite/features/step_definitions/command_steps.rb

```diff
@@ -56,7 +56,8 @@
     # TODO: move that internal resource to some other external location
     STDERR.puts 'Sanity check not implemented, move resource to external network first'
```

Comment:
That's what I suggested during standup: if we get a server in the Uyuni case,then this is not "not implemented anymore". In that case, just remove those lines (the whole "uyuni" case).

Reply:
done